pkg/kubelet/dockertools/docker_manager.go

```diff
@@ -993,19 +993,26 @@ func (dm *DockerManager) getSecurityOpt(pod *api.Pod, ctrName string) ([]string,
 		return nil, nil
 	}
 
+	// in Docker 1.10 the operator is :, for anything later it's =
+	optFmt := "%s=%s"
+	if result <= 0 {
```

Comment:
Both are equivalent due to the check above. Can change it into == 0, makes
it clearer.

On Tue, Jun 28, 2016 at 3:53 PM, Avesh Agarwal <notifications@github.com>
wrote:

> In pkg/kubelet/dockertools/docker_manager.go
> <https://github.com/kubernetes/kubernetes/pull/28159#discussion_r68759801>
> :
>
> > @@ -993,19 +993,26 @@ func (dm *DockerManager) getSecurityOpt(pod *api.Pod, ctrName string) ([]string,
> >  		return nil, nil
> >  	}
> >
> > +	// in Docker 1.10 the operator is :, for anything later it's =
> > +	optFmt := "%s=%s"
> > +	if result <= 0 {
>
> Not sure I am missing something, but above you return when result < 0. Now
> you are checking result <=0. Do you mean just result = 0 here?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/28159/files/40be5e12fb489812a819384953c93665de9d5954#r68759801>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAskC-7zR2dt9W_a0lQ7hpVcJTGpJZevks5qQSdIgaJpZM4JAEnd>
> .
>